src/main/java/com/fasterxml/jackson/module/kotlin/SpreadWrapper.java

```diff
@@ -0,0 +1,46 @@
+package com.fasterxml.jackson.module.kotlin;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
Let's not add new dependencies -- I'd prefer this gets removed.

Reply:
This annotation is derived from `org.jetbrains:annotations`.
This dependency is included in `kotlin-stdlib`.

```
[INFO] +- org.jetbrains.kotlin:kotlin-stdlib:jar:1.4.21:provided
[INFO] |  +- org.jetbrains.kotlin:kotlin-stdlib-common:jar:1.4.21:provided
[INFO] |  \- org.jetbrains:annotations:jar:13.0:provided
```

Confirmation was done with the `mvn dependency:tree` command.